scripts/dev/licensetext.py

```diff
@@ -66,7 +66,7 @@
 """
 
 def previous():
-    '''Return the previous license text, last changed September 21, 2017.'''
+    '''Return the previous license text, last changed January 3, 2018.'''
     # Modify the year in the text
     originalYear = '2015'
     currentYear = '2017'
```

Comment:
I meant to ask why these were 2015 and 2017.

Reply:
Yeah, it's a little more complicated than it should be. The 2015 is there because that's what gets replaced in the original text. This function creates the "previous" license text, so in this context the second variable is badly named. It probably should be called `previousCurrentYear`.

Reply:
I'm just confused why these 2 numbers didn't change from the previous version. I'm willing to accept since the CI returned all green.